### DISCUSSION TITLE
How can I make a required SingleChoiceList empty by default?

### DISCUSSION THREAD

**Discussion by storyteller-dev** [ctreffe/alfred#134] [Alfred Q&A] [answered]
I'd like to add a SingleChoiceList that is mandatory, but empty by default.
1. How do I make it empty by default?
2. Will `force_input=True` still work with an empty default value?

**Comment by storyteller-dev** [comment, marked as answer]
You can add an empty default by simply adding an empty string as the first option:

```python
import alfred3 as al

exp = al.Experiment()

exp += al.Page("Demo Page", name="demo1")

exp.demo1 += al.SingleChoiceList(
    "", "Option 1", "Option 2", "Other",
    name="empty_default_list",
    force_input=True
)
```

And yes, as this first option is an empty string without any whitespace, `force_input=True` will still work

**Reply by jobrachem** [reply]
Great answer! You can improve it by providing a full minimal experiment that can be directly copy-pasted for testing.
